Bizyhood/Core.php

```diff
@@ -1816,6 +1826,67 @@ public function search_shortcode($attrs) {
     }
     
     
+    
+    
+    
+    public function businesses_group_request($groupd_id)
```

Comment:
Spyros, core.php has gotten unwieldy. Can you rearrange the code in core, into modules?